include/ola/thread/Thread.h

```diff
@@ -33,18 +33,6 @@
 
 #include <string>
 
-#if defined(_WIN32) && defined(__GNUC__)
-inline bool operator==(const ptw32_handle_t &left,
-                       const ptw32_handle_t &right) {
-  return (left.p == right.p) && (left.x == right.x);
-}
-
-inline std::ostream& operator<<(std::ostream &stream,
-                                 const ptw32_handle_t &handle) {
-  stream << handle.p;
-  return stream;
-}
-#endif
```

Comment:
why can't we use pthread_equal() instead of operator==
thus we can pass pthread_t and forget about ptw32_handle_t

if so, for the operator<< we can use HAVE_WINPTHREAD ifdef and write 
#ifdef HAVE_WINPTHREAD
inline std::ostream& operator<<(std::ostream &stream,
                                const pthread_t &handle) {
 stream << handle;
 return stream;
}
#endif